lib/svg-sprite/shape.js

```diff
@@ -711,6 +711,14 @@ SVGShape.prototype.setNamespace = function(ns) {
                 }
             });
 
+            // Substitute ID references in href attributes
+            select('//@href', this.dom).forEach(href => {
```

Comment:
Maybe this selector should be more specific to select only needed elements? Like [here](https://stackoverflow.com/a/46274695/1420159)